aten/src/ATen/mps/MPSProfiler.h

```diff
@@ -134,18 +134,24 @@ struct CopyInfo : BaseInfo {
     CPU_TO_MPS,
   };
 
-  CopyInfo(const void* Handle, size_t Length, uint64_t Id, bool IsNonBlocking) :
+  CopyInfo(const void* Handle, size_t Length, uint64_t Id, bool IsNonBlocking, bool UsesBlitter) :
            BaseInfo(Type::COPY, Id, uintptr_t(Handle)), kind(Kind::MPS_TO_MPS),
-           length(Length), isNonBlocking(IsNonBlocking) { }
+           length(Length), isNonBlocking(IsNonBlocking), usesBlitter(UsesBlitter) { }
 
   Kind kind;
   size_t length;
   bool isNonBlocking;
```

Comment:
Should we combine these Booleans to a bitfield ? Do you anticipate more changes like this coming ?